Bacpac/Restore-BcDatabaseFromArtifacts.ps1

```diff
@@ -36,7 +38,8 @@ function Restore-BcDatabaseFromArtifacts {
         [Parameter(Mandatory=$false)]
         [string] $bakFile,
         [switch] $multitenant,
-        [switch] $async
+        [switch] $async,
+        [int] $sqlTimeout = 300
```

Comment:
I cannot find the right value of the default timeout.
The documentation says "none", meaning that it might be up to the server, so we should probably avoid adding the parameter if not specified.
Like setting the default to -1 here and then modify the code for calling New-NavDatabase to:

```
$params = @{
    "DatabaseServer" = $databaseServer
    "DatabaseInstance" = $databaseInstance
    "DatabaseName" = $dbName
    "FilePath" = $bakFile
    "DestinationPath" = $destinationPath
}
if ($sqlTimeout -ge 0) {
    $params += @{ "timeout" = $sqlTimeout }
}
New-NavDatabase @params | Out-Null
```

That would ensure that the timeout isn't set if not provided.

Make sense?